PROTOCOL.md

```diff
@@ -86,10 +86,10 @@ The protocol in general is based on the OBS Remote protocol created by Bill Hami
       - ["GetCurrentScene"](#getcurrentscene)
       - ["GetSceneList"](#getscenelist)
       - ["GetSpecialSources"](#getspecialsources)
-	  - ["GetTextGDIPlusProperties"](#gettextgdiplusproperties)
```

Comment:
It was an issue with tab width (in space chars). Maybe your IDE/text editor compensated for it.